tests/ha/test_control_pod_restart.py

```diff
@@ -112,13 +123,19 @@ def setup_method(self):
         self.restore_node = False
         self.deploy = False
         self.s3_clean = dict()
+        self.restore_pod = None
         LOGGER.info("Check the overall status of the cluster.")
         resp = self.ha_obj.check_cluster_status(self.node_master_list[0])
         if not resp[0]:
             resp = self.ha_obj.restart_cluster(self.node_master_list[0])
             assert_utils.assert_true(resp[0], resp[1])
         LOGGER.info("Cluster status is online.")
-        self.restore_pod = self.restore_method = None
+        self.s3acc_name = "{}_{}".format("ha_s3acc", int(perf_counter_ns()))
+        self.bucket_name = "ha-mp-bkt-{}".format(self.random_time)
```

Comment:
can we use one of them? self.random_time or perf_counter_ns() or they are used diff intentionally?

Reply:
better we avoide use of self.random_time as it will set one time   in setup method. But we don't want that. 
Use perf_counter_ns() will always give some new digits combination.

Reply:
perf_counter_ns() will be useful if we are creating strings in loop, for single string we can use random time.